sw/device/lib/dif/dif_otp_ctrl.c

```diff
@@ -427,6 +427,29 @@ static const partition_info_t kPartitions[] = {
         },
 };
 
+dif_result_t dif_otp_ctrl_relative_address(const dif_otp_ctrl_t *otp,
```

Comment:
nit: for consistency, matching unit tests go in `dif_otp_ctrl_unittest.cc`, though not that vital for now

Reply:
since your manuf tests will test this too presumably :)

Reply:
Done!